### DISCUSSION TITLE
Constructive Conflict in Conscious Coliving - what do you think?

### DISCUSSION THREAD

**Discussion by laurenwigmore** [life-itself/community#890] [Ideas]
This discussion thread is following on from our recent poll where we asked what topics would you like in our follow up course to [Conscious Coliving 101](https://lifeitself.org/conscious-coliving-course). As a result of this discussion here is our draft skeleton for our *Constructive Conflict in Conscious Coliving* email course. 

We'd love further ideas and input to help us shape it!

**1** Setting the context on conflict:
- Conflict happens - that’s life: brief insight into sociological theories, and role of biology and psychology. Before setting it up as a normal occurrence in everyday life and most relationship dynamics.
- Conflict can be an awesome opportunity for growth
- The two sides of the coin for conscious coliving
    - Conflict may happen more because you are in a community (i.e. you can avoid your mum by not talking to her. We can avoid (obvious) conflict by living on our own) - but living in a community offers us the opportunity to see it as more of a chance to practice and grow!
    - Conscious community provides support in situations of conflict
- When not managed conflict can blow up a community

**2** Context 2: conflict and the inability to see others perspective is at the root of so much of our suffering …
- Labeling etc
- The roots of conflict are always emotional/belief based
- The difference between healthy vs toxic conflict
- Internal conflict also adds to our suffering

**3** Ontological capacity basis (aka emotional intelligence) - will be addressed across 2 or 3 episodes	
- The kinds of capacities we are talking about especially 2nd person perspective
   - Piaget experiment with kids video
   - Maybe we can take a “physical” perspective but many of us struggle to be able to take a “view” perspective (which is the source of so much conflict)
- Information on/how to develop these capacities personally
- How to develop collectively e.g. Non Violent Communication (NVC), expressing our needs skillfully (e.g. Your music is way too loud, turn it down vs “I find it difficult to concentrate…)

**4** Conflict resolution processes (formal)
- There are actually a lot of options. But here are a few curated examples that we recommend
    - To EMPHASIZE: process is really secondary to capacity. No process will work without the capacity to listen, to limit activation, to act in good faith etc
 - How to begin anew (? from PV)
 - Consult/utilize the expertise and work of Karl Steyaert (to confirm with Karl

**Comment by martinprogrammer** [comment]
@laurenwigmore  Very well done, enjoyed reading through!  :-)

**Reply by rufuspollock** [reply]
@martinprogrammer have you signed up for the course now the course is live?

**Reply by martinprogrammer** [reply]
@rufuspollock Yep, this was my initial feedback.

**Reply by laurenwigmore** [reply]
That's great, thanks Martin! Appreciate the feedback.
